src/CovidData/Card/index.tsx

```diff
@@ -1,12 +1,132 @@
 import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import {
+  ActivityIndicator,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from "react-native"
+import { useNavigation } from "@react-navigation/native"
 
+import { HomeStackScreens } from "../../navigation"
+import { Text } from "../../components"
+import SectionButton from "../../Home/SectionButton"
+import { CovidDataRequest } from "../Context"
 import { useCovidDataContext } from "../Context"
-import CovidDataCard from "./CovidDataCard"
+import CovidDataInfo from "./CovidDataInfo"
+
+import {
+  Iconography,
+  Affordances,
+  Colors,
+  Typography,
+  Spacing,
+} from "../../styles"
 
 const CovidDataCardConnector: FunctionComponent = () => {
   const { request, locationName } = useCovidDataContext()
 
   return <CovidDataCard dataRequest={request} locationName={locationName} />
 }
 
+type CardContainerProps = {
```

Comment:
ended up inlining this component as the connector component is largely a wrapper around this main content card anyways.